src/util/createEmptyProject.js

```diff
@@ -0,0 +1,15 @@
+const fs = require("fs");
+const titleize = require("../util/titleize");
+
+const cleanupAndTitleize = (str) => {
+  const cleaned = str.replace(/[-_]/g, " ");
```

Comment:
The logic on line 5 is duplicated in cleanupAndCapitalize.js. Should this be extracted as another utility?

Reply:
Yeah, it could be. Perhaps then we could name it `cleanupProjectName` or `cleanupWorkflowName`. Then we wouldn't need `cleanupAndTitleize`/`cleanupAndCapitalize`--we could just nest function calls like `titleize(cleanupProjectName(string))` rather than `cleanupAndTitleize(string)`.